src/utils/formatters/Formatters.tsx

```diff
@@ -92,3 +92,11 @@ export const Money = (props: FormatAmountProps) => {
     <Currency value={otherProps.token}/>
   </span>);
 };
+
+export const FormatQuoteToken = (props: { token: string }) => {
```

Comment:
I would change it to something else. You are not formatting it. Not quite sure if we need it extract actually.

Reply:
I am. https://www.quora.com/What-is-text-formatting
This is a perfect candidate for extraction. Coloured token names can appear in other places.

Reply:
Can appear but not right now. Also I showed you that you can do it all with CSS. No need to add additional JS for that

Reply:
The point of using the component is that it can be easily reused in other places.
The CSS approach you are suggesting below does not make it reusable.

Reply:
Actually it makes it pretty reusable. 
if you have :global(.colorfulDaiText){ color: $... } and everywhere you apply it it will work. I mean you can leave it as it it. Commit comments are just suggestions right :)

Reply:
The current approach is the simplest one.
If we add more occurrences of this kind of colouring and we find a better approach, then we might refactor.